Master-Game/World/Map.cs

```diff
@@ -39,5 +53,91 @@ public void ResetTiles()
                 }
             }
         }
+
+        public MapData LoadJson()
+        {
+            string fileName = "Jay01.json";
+            string filePath = Path.GetDirectoryName(Path.GetDirectoryName(System.IO.Directory.GetCurrentDirectory())) + "\\Assets\\Maps\\" + fileName;
+            FileInfo f = new FileInfo(fileName);
```

Comment:
Prefer not to use single letter variables. Call this fileInfo

Reply:
Yeah, sorry about this. That should have been removed. I do not use this at all. I removed it.